### PR TITLE
Added taming script

### PR DESCRIPTION
**Pull request by EPuncker**
it was added on 8.7 version and updated with mounts until 10.98 version

- added missing article to scorpion sceptre

script by ORTS

### CODE REVIEW

**Comment by BahamutxD** [comment]
Tin Lizzard and Uniwheel (all TYPE_ITEM) are not working.

Rest work just fine.

Just incase - missing https://tibia.fandom.com/wiki/The_Lion%27s_Heart not sure if on purpose? Music box could be added here too I guess.

**Comment by EPuncker** [comment]
> Tin Lizzard and Uniwheel (both TYPE_ITEM) are not working. Donkey either works.
> 
> Rest work just fine.
> 
> Just incase - missing https://tibia.fandom.com/wiki/The_Lion%27s_Heart not sure if on purpose?

indeed it was missing, thanks for noticing! I've added it, and about the donkey it is working with the changes from #2981 , now the only missing thing is to figure out why the item_type is not working

**Comment by DSpeichert** [comment]
@EPuncker I've converted to draft since it sounds like something isn't working yet.

**Comment by BahamutxD** [comment]
Changing first line to 

`local TYPE_ITEM, TYPE_MONSTER, TYPE_NPC = 1, 2, 3`

fix the issue with ITEM mounts although it will throw an error

`taming.lua:379: attempt to call method 'say' (a nil value)`

fix seen here https://otland.net/threads/10-77-tfs-1-2-orts-a-real-map-project.204514/post-2327652

Donkey mount works perfect now just like the rest.

**Comment by EPuncker** [comment]
okay, everything seems to be working fine now, can you check please @BahamutxD ?

**Comment by BahamutxD** [comment]
@EPuncker Everything seems to be working just fine now including achievements.

File: data/scripts/actions/others/taming.lua [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+		type = TYPE_MONSTER,
+		achievement = "Gear Up",
+		chance = 20,
+		fail = {
+			{run = true, sound = "*click clack*", text = "The walker disappeared."},
+			{broke = true, text = "Aah no! The glooth batteries of this unit must have leaked."},
+			{sound = "*pling*", text = "The walker seems incompatible with this control unit."}
+		},
+		success = {sound = "*brzzz*", text = "You tamed the walker."}
+	},
```
Comment (M4g1ch) [inline data/scripts/actions/others/taming.lua:350]: There is no such monster within monster files

File: data/scripts/actions/others/taming.lua [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+		id = 40,
+		type = TYPE_MONSTER,
+		achievement = "Lion King",
+		chance = 20,
+		fail = {
+			{run = true, text = "The noble lion fled."},
+			{sound = "GRRRRRRRRR", text = "The noble lion majestically rejects your amulet."}
+		},
+		success = {sound = "Grrr", text = "The noble lion will now accompany you as a friend and ally."}
+	}
```
Comment (M4g1ch) [inline data/scripts/actions/others/taming.lua:362]: There is no such monster within monster files

**Comment by EPuncker** [comment]
> Maybe it's not a problem as this should work when those missing monsters are added

true thanks for reminding me, I'll add them in a separated PR

**Comment by BahamutxD** [comment]
If I'm not mistaken this is just missing Rift Runner mount now, the rest came from Premium Shop.

https://tibia.fandom.com/wiki/Mysterious_Scroll
https://tibia.fandom.com/wiki/Rift_Runner

File: data/scripts/actions/others/taming.lua [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+		success = {text = "You receive the permission to ride a neon sparkid."}
+	},
+	[26341] = { -- menacing egg
+		name = "vortexion",
+		id = 99,
+		type = TYPE_MONSTER,
+		achievement = "Vortex Tamer",
+		chance = 100,
+		success = {text = "You receive the permission to ride a vortexion."}
+	}
```
Comment (M4g1ch) [inline data/scripts/actions/others/taming.lua:386]: There are no such monsters within monster files

Comment (M4g1ch) [reply data/scripts/actions/others/taming.lua:386]: After testing those, I'll approve :smile:

**Comment by EPuncker** [comment]
> If I'm not mistaken this is just missing Rift Runner mount now, the rest came from Premium Shop.
> 
> https://tibia.fandom.com/wiki/Mysterious_Scroll
> https://tibia.fandom.com/wiki/Rift_Runner

wow I missed this one, never heard of it, and now that I searched youtube, couldn't find any video taming it, I'll assume one you use the Mysterious_Scroll you get the mount?

File: data/scripts/actions/others/taming.lua (hunk 3 above)
Comment (EPuncker) [reply data/scripts/actions/others/taming.lua:386]: I'll add them soon, don't worry 😛

**Pull request merged**